### ISSUE TITLE
Instrument name in export file name

### ISSUE THREAD

**Issue by cla023** [BjerknesClimateDataCentre/QuinCe#611]

**Comment by squaregoldfish** [comment]
The filename should contain the data set name (i.e. EXPO Code) as well. There are several options here - perhaps it should be customisable at download time?

**Comment by cla023** [comment]
I agree that some kind of dataset identifier is also necessary. Customisable names might be usefull too, and there could be different default names for the different export purposes (names for CP are different than names for SOCAT).
